Fix:

warning: comparison of array 'tmp_nickname' equal to a null pointer is always false [-Wtautological-pointer-compare]
--- server.cc.orig	2004-04-26 12:41:49.000000000 -0500
+++ server.cc	2024-10-16 14:32:21.000000000 -0500
@@ -1044,7 +1044,7 @@
 	/* :nickname!pcname@addr QUIT :"leave-message" */
 	char tmp_nickname[NICKNAME_LEN];
 	GetSenderName(tmp_nickname, 0, NICKNAME_LEN);
-	if(tmp_nickname == '\0') return; /* false format */
+	if(tmp_nickname[0] == '\0') return; /* false format */
 	/* get the quit message */
 	Put_Text_together(2);
 	if(normal_text[0] == '\0') {
